support/cas-server-support-oauth/src/test/java/org/apereo/cas/support/oauth/logout/OAuth20LogoutReplicateSessionTests.java

```diff
@@ -0,0 +1,43 @@
+package org.apereo.cas.support.oauth.logout;
+
+import org.apereo.cas.AbstractOAuth20Tests;
+import org.apereo.cas.logout.LogoutExecutionPlan;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.EnableAspectJAutoProxy;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * This is {@link OAuth20LogoutReplicateSessionTests}.
+ *
+ * @author Jerome Leleu
+ * @since 6.5.0
+ */
+@SpringBootTest(classes = AbstractOAuth20Tests.SharedTestConfiguration.class,
+        properties = {
+                "cas.authn.attribute-repository.stub.attributes.uid=cas",
+                "cas.authn.attribute-repository.stub.attributes.givenName=apereo-cas",
+                "spring.main.allow-bean-definition-overriding=true",
+                "cas.authn.oauth.replicate-sessions=true"
+        })
+@EnableTransactionManagement
+@EnableAspectJAutoProxy
+@Tag("OAuth")
+public class OAuth20LogoutReplicateSessionTests {
+
+    @Autowired
+    @Qualifier("logoutExecutionPlan")
+    private LogoutExecutionPlan logoutExecutionPlan;
+
+    @Test
+    public void verifyThatTheOAuthSpecificLogoutPostProcessorIsRegistered() throws Exception {
```

Comment:
This is not a very valid test. You need to actually ensure the session is removed; not just that it's registered into the plan.

Reply:
Yes, I focused on the registration, but indeed, I will augment the test to ensure the session is actually removed.

Reply:
The test has been completed to ensure that the distributed session is actually destroyed when the logout is called.

I also added the two new test classes to the suite (something I forgot).

So everything should be ok for the merge.